sdk/core/core-lro/src/http/operation.ts

```diff
@@ -304,15 +304,20 @@ export function getOperationStatus<TState>(
   }
 }
 
+function accessBody<P extends string>(
```

Comment:
Not sure the name accessBodyProperty would be more proper, accessBody seems getting the body content?